Userland/Libraries/LibWeb/XHR/FormData.idl

```diff
@@ -1,6 +1,10 @@
 #import <FileAPI/Blob.idl>
 #import <FileAPI/File.idl>
-#import <HTML/HTMLFormElement.idl>
+// FIXME: This #import currently gives the following error after XHR/FormData.idl was #imported in Fetch/BodyInit.idl:
+//        "LibWeb/HTML/Window.idl:114: error: Mixin 'WindowOrWorkerGlobalScope' was never defined."
+//        XHR/FormData.idl needs to be #imported in Fetch/BodyInit.idl while removing #import HTML/HTMLFormElement.idl
+//        currently makes no difference.
+// #import <HTML/HTMLFormElement.idl>
```

Comment:
I don't understand how that's related to this commit? It merely adds a new function and should have no effect on these IDL files...?

Reply:
Ooops, moved this to the correct commit.